### PR TITLE
Bumping to new version of GeoServer (2.8.1) and GeoFence (3.0)

### PR DESCRIPTION
**Pull request by pmauduit**
The purpose of this PR is to integrate evolutions proposed in GeoServer / GeoFence upstream into geOrchestra.

This PR is mainly related to pom / doc / submodule modifications and does not contain so much code, since everything is already published into the GeoServer / GeoFence fork of geOrchestra (see https://github.com/georchestra/geoserver/tree/2.8.1-layergroups-georchestra for geoserver, https://github.com/georchestra/geofence/tree/3.0-georchestra for geofence standalone).

Note: there is still a buggy page in the GS interface (the "about geoserver" page is broken).

### CODE REVIEW

File: geoserver/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+$ mvn clean install
+```
+
+### GeoServer with GeoFence, without the internal UI
+
+```
+$ mvn clean install -Pgeofence
+```
+
+### GeoServer with GeoFence and the internal UI
```
Comment (fvanderbiest) [inline geoserver/README.md]: GeoServer with integrated GeoFence UI

File: geoserver/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### GeoServer without GeoFence
+
+```
+$ cd geoserver-submodule/src
+# without GeoFence:
+$ mvn clean install
+```
+
+### GeoServer with GeoFence, without the internal UI
```
Comment (fvanderbiest) [inline geoserver/README.md]: GeoServer with GeoFence's own UI

**Comment by fvanderbiest** [comment]
Thanks Pierre.
We must not forget to make the target branches in the submodules the default ones affter this is merged.

Also, could you rename branch 2.8.1-layergroups-georchestra into 2.8.1-georchestra ?

File: geoserver/README.md (hunk 1 above)
Comment (pmauduit) [inline geoserver/README.md]: done

File: geoserver/README.md (hunk 2 above)
Comment (pmauduit) [inline geoserver/README.md]: done

**Comment by pmauduit** [comment]
> Also, could you rename branch 2.8.1-layergroups-georchestra into 2.8.1-georchestra ?

I assume you meant into the geoserver submodule. I began to do it onto the current PR ...

**Pull request closed without merging**